src/main/kotlin/edu/ics499/team1/app/entities/UserEntity.kt

```diff
@@ -0,0 +1,19 @@
+package edu.ics499.team1.app.entities
+
+import com.fasterxml.jackson.annotation.JsonManagedReference
+import jakarta.persistence.*
+
+@Entity
+@Table(name = "users")
+data class UserEntity(
+    @Id
+    @GeneratedValue
+    val userId: Int = 0,
+    val firstName: String,
+    val lastName: String,
+    val email: String,
+    val phoneNumber: String?,
+    @OneToMany(mappedBy = "user", cascade = [CascadeType.ALL])
+    @JsonManagedReference
```

Comment:
When we have a ManyToOne we also need to set the OneToMany relationship. 

Setting @JsonManagedReference allows us to pull the whole vehicle list without failing. We also need to set the @JsonBackReference in the Vehicle entity under the ManyToOne relationship.

Essentially these will be used when we need to pull a list of something from another table.